packages/tabs/test/scroll.test.js

```diff
@@ -64,6 +72,96 @@ describe('scrollable tabs', () => {
       btn.click();
       expect(scroller.scrollLeft).to.be.equal(0);
     });
+
+    ['ltr', 'rtl'].forEach((dir) => {
+      describe(dir, () => {
+        beforeEach(async () => {
+          tabs.setAttribute('dir', dir);
+          await nextFrame();
+        });
+
+        it('should display the next item first on forward button click', async () => {
+          const forwardButtonWidth = tabs._getNavigationButtonVisibleWidth('forward-button');
```

Comment:
In general it's not good practice to reuse implementation for test setups and assertions. It's also a bit difficult to figure out what's going on in this test.

Given that the original issue was that some tabs were never or only partially visible, a more comprehensive and maybe easier test setup would be:
- click forward/backward until scroll position doesn't change anymore
- verify that each tab was fully visible at some point

Reply:
Updated the tests similar to your suggestion. It is hard to avoid the similar setup completely though, since even in this case we have to check their visibility.